documentation/source/refcard.rst

```diff
@@ -0,0 +1,93 @@
+.. _refcard:
+
+**************
+Reference Card
+**************
+..
+   Document "best practice"; leave out detail.
+   Keep format as 1/3rd width of A4/Letter for taping to monitor frames.
+
+   The "| " syntax is a "Line Block", see
+      https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#line-blocks
+
+*coro*: a coroutine; *task*: a running coroutine
+
++------------------------+----------------------------------------------------------------+
+| Assign                 | ``dut.mysignal <= 0xFF00``                                     |
++------------------------+----------------------------------------------------------------+
+| Assign immediately     | ``dut.mysignal.setimmediatevalue(0xFF00)``                     |
++------------------------+----------------------------------------------------------------+
+| Assign metavalue       | ``dut.mysignal <= BinaryValue("X")``                           |
++------------------------+----------------------------------------------------------------+
+| Read                   | | ``val = dut.mysignal.value``                                 |
+|                        | | (``mysig = dut.mysignal`` *creates an alias/reference)*      |
++------------------------+----------------------------------------------------------------+
+| Bit slice              | | ``mybit = dut.mysignal[0].value``                            |
+|                        | | ``mybits = dut.mysignal[1:0].value``                         |
```

Comment:
I'm not sure slicing the signal always works. You might want to move it after the `.value`

Reply:
Is this not an issue?

Reply:
`dut.mysignal[0].value` should work if it is an array, so I'll use a different name to make it clearer and have `dut.mysignal.value[0]` in an extra line.
`[1:0]` may indeed not work at all.

Reply:
Done.